Dockerfile

```diff
@@ -3,8 +3,16 @@ FROM openjdk:8u171-jre-alpine
 ARG kafka_version=2.0.0
 ARG scala_version=2.12
 ARG glibc_version=2.27-r0
-
-MAINTAINER wurstmeister
+ARG vcs_ref=unspecified
+
+LABEL org.label-schema.name="kafka" \
+      org.label-schema.description="Apache Kafka" \
+      org.label-schema.vcs-url="https://github.com/wurstmeister/kafka-docker" \
+      org.label-schema.vcs-ref="${vcs_ref}" \
+      org.label-schema.schema-version="1.0" \
+      maintainer="wurstmeister" \
+      org.apache.kafka.scala-version="${scala_version}" \
```

Comment:
Given that this is not an offiically supported apache.org image, it might be better to use the label-schema suggested label of `version` and remove the `org.apache.xxx` labels i.e.

```
org.label-schema.version="${scala_version}_${kafka_version}"
```

This also matches the tag of the image build on dockerhub.